Themes/default/css/index.css

```diff
@@ -1688,7 +1688,7 @@ tr.windowbg td, tr.windowbg2 td, .table_grid tr td {
 .generic_icons.invalid {
 	background-position: -109px -5px;
 }
-.generic_icons.gender_Female {
+.generic_icons.gender_2 {
```

Comment:
If no changes are made to the logic where None is skipped because it is a special value then this should be changed from `.generic_icons.gender_2` to `.generic_icons.gender_1`.